sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,25 +1,45 @@
 # Release History
 
-## 1.8.0-beta.4 (Unreleased)
+## 1.8.0 (2023-02-02)
 
 ### Features Added
 
+- Added support for parsing space character in place of 'T' in RFC3339 DateTimes.
+- Added support for HTTP proxy servers, both unauthenticated and with Plain authentication.
+- Added universal support for several TLS options:
+  - Added the ability to set the expected TLS root certificate for TLS connection (useful if a proxy server uses a TLS certificate that is not chained to a known root).
+  - Added the ability to enable TLS certificate revocation list checks (off by default).
+    - For libcurl only: Allow TLS connection to succeed if CRL retrieval fails.
+    - *NOTE*: This change only applies if libcurl is built using the OpenSSL crypto backend. It does NOT apply if libcurl uses the schannel (Windows default) or SecureTransport (macOS/iOS default).
+
 ### Breaking Changes
 
```

Comment:
```suggestion
- Changed the name of several HTTP span attributes generated by the `RequestActivityPolicy`:
  - `requestId` is renamed to `az.client_request_id`
  - `serviceRequestId` is renamed to `az.service_request_id`
```

Reply:
`RequestActivityPolicy` is in `_internal` namespace, why are we listing these changes here? How are they visible to the customer who does not consume any type from the `_internal` or `_detail` namespace?

Reply:
Agreed, internal only changes don't need to be highlighted in the CL unless absolutely necessary, to reduce noise to signal ratio.

Reply:
Maybe we can put it as "Changed the name of several telemetry span attributes"?

i.e. without mentioning the policy. But users will see attributes emitted with different names. Do I understand it correctly

Reply:
THese are publicly visible changes - OpenTelemetry attributes can be read by an OpenTelemetry exporter.